addons/languages/coptic/apk/src/main/play/listings/en-US/full-description.txt

```diff
@@ -1,10 +1,9 @@
-Coptic Language pack
+Coptic Language input for AnySoftKeyboard
```

Comment:
Is there a benefit for using `input` rather than `pack`?

Reply:
No. It seemed more descriptive, but I'm happy to use *pack* if you prefer.

There's a lot of variation among the various language addons. The most common seem to be `$langauge layouts and dictionary [for AnySoftKeyboard]` and `$language Language pack` Do you have a preference between those?

Reply:
Maybe `$language Language pack` is better, as it a more generic term.

Reply:
could you change that? And also rebase the branch

Reply:
Ok, description changed and rebased against master.